src/musicbrainz/web/musicbrainzrecordingstask.cpp

```diff
@@ -26,6 +26,10 @@ const QString kRequestPath = QStringLiteral("/ws/2/recording/");
 
 const QByteArray kUserAgentRawHeaderKey = "User-Agent";
 
+// MusicBrainz allows only a single request per second on average
+// to avoid rate limiting. Otherwise requests fail with 503.
+constexpr Duration kMinDurationBetweenRequests = Duration::fromMillis(1000);
```

Comment:
> Otherwise requests fail with 503.

I don't think that is the case, did you check it? 
From experience it is 403 or no responds at all.

Reply:
https://musicbrainz.org/doc/MusicBrainz_API/Rate_Limiting:

> and decline (http 503) the rest